population/create_glpi_computer_redfish.py

```diff
@@ -1043,7 +1045,11 @@ def add_rack_location_from_sunbird(
                 "No Sunbird config provided, setting data center and room automatically"
             )
             data_center = location_data["tiDataCenterName"]
-            room = location_data["tiRoomName"]
+            if "tiRoomName" in location_data:
+                room = location_data["tiRoomName"]
+            else:
+                # If the Room is not specified in Sunbird, use the Data Center name
+                room = location_data["tiDataCenterName"]
```

Comment:
Any handling needed for a case where that's also not populated? Or is that a case that isn't really likely?

Reply:
Changed `data_center` to default to `None` if tiDataCenterName doesn't exist, and added an elif to check this.

Reply:
@adhil0 Got it, makes sense